hsakmt-roct/PKGBUILD

```diff
@@ -1,62 +1,34 @@
-# Maintainer: Olaf Leidinger <oleid@mescharet.de>
+# Maintainer: acxz <akashpatel2008 at yahoo dot com>
+# Contributor: Jakub Okoński <jakub@okonski.org>
+# Contributor: Olaf Leidinger <oleid@mescharet.de>
+# Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
+
 pkgname=hsakmt-roct
-pkgver=1.8.0.r0.e3dd067
-pkgrel=2
+pkgver=3.1.0
+pkgrel=1
 pkgdesc="Radeon Open Compute Thunk Interface"
-_gitdir=ROCT-Thunk-Interface
 arch=('x86_64')
-url="https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface"
-license=('X11/MIT')
-groups=()
-depends=(pciutils numactl)
-makedepends=(git cmake gcc) 
-provides=("${pkgname%-git}")
-conflicts=("${pkgname%-git}")
-replaces=()
-backup=()
-options=()
-source=('git+https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface.git#branch=roc-1.8.x')
-md5sums=('SKIP')
-
-pkgver() {
-	cd "$srcdir/${_gitdir}"
-
-	# Git, tags available
-	printf "%s" "$(git describe --long | sed -e 's:roc-::g'  -e 's/\([^-]*-\)g/r\1/;s/-/./g')"
-}
+url='https://github.com/RadeonOpenCompute/ROCT-Thunk-Interface'
+license=('MIT')
+depends=('numactl' 'pciutils')
+makedepends=('cmake')
+provides=("roct-thunk-interface=$pkgver")
+conflicts=('roct-thunk-interface')
```

Comment:
do we need the provides and conflicts here? I think the replaces should suffice.
We will have to get rid of `roct-thunk-interface` everywhere it is listed as a dep, of course.
Do you think we should just do it in this PR?

Reply:
Hmm, there's no need, but it might be better to do so.

Reply:
Yeah let's do it in this` PR since we want to deprecate `roct-thunk-interface` anyway.`

Reply:
Arch wiki is unclear about `provides` vs `conflicts`, it states that two packages with the same `provides` package will conflict with each other but it does not state that, in this case, `hsakmt-roct` will trigger a conflict with `roct-thunk-interface`.